### DISCUSSION TITLE
Collections are not visible within organization view when adding a new item

### DISCUSSION THREAD

**Discussion by jimmyparr** [dani-garcia/vaultwarden#2331] [Q&A]
vaultwarden/latest
bug from 2 mouth ago 
here a video how to reproduce the bug

https://ordifax.com/VAULTWARDEN/IMG_3535.MOV

**Comment by BlackDex** [comment]
I actually think this is the same as #2290, if that is the case, it looks like it is something with Bitwardens web-vault.

Also as a reference since i think this is a continuation of #2153.
Where i asked for more details the is provided here.

If it is the same as #2290, it's not something on the Vaultwarden side.

**Comment by jimmyparr** [comment]
this bug dows not occur in version 1.23 and before

**Comment by jimmyparr** [comment]
and yes 2153 it me that open the ticket

**Comment by BlackDex** [comment]
Then, I think this is a bug in the web-vault.
Please try to reproduce this on vault.bitwarden.com, it allows an organization with two members so you should be able to try it out there. If that is the case, it is either a bug, or how they designed it, that is why i think this is the same as #2290.

Version 1.23.0 uses web-vault 2.23.0, 1.23.1 uses web-vault 2.25.0

**Comment by jimmyparr** [comment]
you cannot test on bitwarden because the bug only occur if you have more
than 2 organization

single organisation are not affected

free account only support 1 org

ans as i mention 2 mounth ago

the bug is not present on the first organisation in the list only the all
other one!

if i revove the 1 organisation the next first organisation does not have
the bug only the remaining and so on

**Comment by BlackDex** [comment]
Then please try to write a step-by-step description including creating organizations, collections and users and which rights they need to have. And then how to trigger this exactly, because I'm not able to reproduce this as i mentioned before. Hopefully with a very detailed step-by-step explanation I'm able to reproduce this.

**Comment by jimmyparr** [comment]
i mean look at my video

- organisation as to be set as non personal use
-only thing can be add in collectiion

do this for 2 organisation, and put youself as user withing one of those
organization!

look the video the bug is reproduce in 30 seconde

**Comment by jimmyparr** [comment]
anyway close this ticket ! a found my solution by myself

any user as to be admin or owner else it will broke

the goal was to give the solution

don’t bother reply

thank’s

**Comment by jimmyparr** [comment]
make it clear !

all user as to be admin or greater in all the ORG they are in , to not have
any issue.

in any user have only one ORG where he is manager or user the bug will
occur in all ORG EXCEPT DE FIRT ONE IN THE LIST

- ALL ORG are collection only add stuff allowed( no personal.
- disable send
- 2FA
- strogn password (choice 4)
- no ORG set as businnes
- no user set as only for that ORG

**Comment by jimmyparr** [comment]
hope what i found will help other

peace ya

**Comment by jimmyparr** [comment]
OK STEP TO REPRODUCE FROM A-Z WITH A BRAND NEW VW_DATA - EVERYTHING FRESH
CREATE 3 ORG LIKE THIS

https://ordifax.com/VAULTWARDEN/A.jpg
https://ordifax.com/VAULTWARDEN/B.jpg
https://ordifax.com/VAULTWARDEN/C.jpg
https://ordifax.com/VAULTWARDEN/D.jpg
https://ordifax.com/VAULTWARDEN/E.jpg
https://ordifax.com/VAULTWARDEN/F.jpg
https://ordifax.com/VAULTWARDEN/VIDEO.MOV

take a closer look at warning in picture C
it mean, that user and manager are affected by the policy enforcement, but then it brake the fact to add item in collection in ORG.
so by adding user as admin or owner, it screw my policy for personal ownership!

I notice from the begenning that if you do not select any ORG, go on your vault, from there you can select your right ORG AND COLLECTION, from the main menu ! PICTURE F


ET VOILÀ

**Comment by BlackDex** [comment]
@jimmyparr, thanks for your patients and the later posts, images and video. With this information i was able to reproduce the issue not only with Vaultwarden, but also with Bitwarden Selfhosted. It looks like this bug sneaked in somewhere between v2.23.0 and v2.24.0 of the Bitwarden web-vault.

Because this is not something specific to Vaultwarden i have created a ticket with the web-vault repository which you can few here: https://github.com/bitwarden/web/issues/1499

Because this isn't something we can fix on our side (I tried to look for the issue, but TypeScript isn't really my strongest language), i will close this ticket and convert it to a discussion.
